spec/models/category_spec.rb

```diff
@@ -35,3 +30,11 @@ def assign_videos_to_category(count, category)
     expect(comedy.videos).to eq []
   end
 end
+
+private
```

Comment:
This certainly isn't the right use for `private` under what class it makes it private method ?